Documentation/CrossPlatform/OculusQuestMRTK.md

```diff
@@ -71,7 +71,7 @@ In that config file, you should see the HandTrackingSupport field set to "Contro
 
         ![OculusAddXRSDKDataProvider](../Images/CrossPlatform/OculusQuest/OculusAddDataXRSDKProvider.png)
 
-    - You can verify that the Oculus Controllers are detected by 
+1. The Oculus XRSDK Data Provider includes a OVR Camera Rig Prefab which automatically configures the project with an OVR Camera Rig and OVR Hands to properly route input. Adding your own OVR Camera Rig to the scene will require you to manage your settings and input manually.
```

Comment:
Also a suggestion to try to replace things like "your own" with "a custom" or something, to align with our documentation guideline "try to avoid 'you'":

https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#writing-style